### PR TITLE
Utilities improvement

### PR DESCRIPTION
**Pull request by christophwelling**
Makes NuRadioMC use the units and fft utilities from NuRadioReco.
It also already implements the changes in the fft utilities on the NuRadioReco PR https://github.com/nu-radio/NuRadioReco/pull/155

### CODE REVIEW

File: NuRadioMC/SignalGen/HCRB2017.py [hunk 1]
```diff
@@ -92,7 +92,7 @@ def get_frequency_spectrum(energy, theta, N, dt, is_em_shower, n, R, LPM=True, a
         if variable set, the shower width is manually set to this value
     """
     eR, eTheta, ePhi = get_time_trace(energy, theta, N, dt, is_em_shower, n, R, LPM, a=a)
-    return np.array([fft.time2freq(eR), fft.time2freq(eTheta), fft.time2freq(ePhi)])
+    return np.array([fft.time2freq(eR, dt), fft.time2freq(eTheta, dt), fft.time2freq(ePhi, dt)])
```
Comment (cg-laser) [inline NuRadioMC/SignalGen/HCRB2017.py]: the function takes the sampling rate as argument and not dt

File: NuRadioMC/simulation/simulation.py [hunk 2]
```diff
@@ -521,7 +521,7 @@ def run(self):
                             from matplotlib import pyplot as plt
                             fig, (ax, ax2) = plt.subplots(1, 2)
                             ax.plot(self._ff, np.abs(eTheta) / units.micro / units.V * units.m)
-                            ax2.plot(self._tt, fft.freq2time(eTheta) / units.micro / units.V * units.m)
+                            ax2.plot(self._tt, fft.freq2time(eTheta, self._dt) / units.micro / units.V * units.m)
```
Comment (cg-laser) [inline NuRadioMC/simulation/simulation.py]: again here, shouldn't it be `1/self._dt`?

Comment (christophwelling) [reply NuRadioMC/simulation/simulation.py]: Yes, you are right. I fixed it

File: NuRadioMC/SignalGen/HCRB2017.py (hunk 1 above)
Comment (christophwelling) [reply NuRadioMC/SignalGen/HCRB2017.py]: Fixed

**Pull request merged**